ncloud/resource_ncloud_nks_node_pool.go

```diff
@@ -0,0 +1,328 @@
+package ncloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/ncloud"
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/services/vnks"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func init() {
+	RegisterResource("ncloud_nks_node_pool", resourceNcloudNKSNodePool())
+}
+
+const (
+	NKSNodePoolStatusRunCode       = "RUN"
+	NKSNodePoolStatusNodeScaleDown = "NODE_SCALE_DOWN"
+	NKSNodePoolStatusNodeScaleOut  = "NODE_SCALE_OUT"
+	NKSNodePoolIDSeparator         = ":"
+)
+
+func resourceNcloudNKSNodePool() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceNcloudNKSNodePoolCreate,
+		ReadContext:   resourceNcloudNKSNodePoolRead,
+		UpdateContext: resourceNcloudNKSNodePoolUpdate,
+		DeleteContext: resourceNcloudNKSNodePoolDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(DefaultCreateTimeout),
+			Update: schema.DefaultTimeout(DefaultCreateTimeout),
+			Delete: schema.DefaultTimeout(DefaultCreateTimeout),
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_uuid": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"instance_no": {
+				Type:     schema.TypeInt,
```

Comment:
```suggestion
				Type:     schema.TypeString,
```
다른 모듈과 통일성을 위해 위와 같이 변경 해 주실수 있을까요?

Reply:
네 `TypeString` 으로 변경하겠습니다.